app/controllers/avatars.js

```diff
@@ -1,7 +1,7 @@
 /**
  * List of Avatars
  */
-avatars = ['/img/chosen/E01.png',
+const avatars = ['/img/chosen/E01.png',
            '/img/chosen/F01.png',
            '/img/chosen/FA04.png',
            '/img/chosen/FB03.png',
```

Comment:
Please ensure that you use one `const` or `let` declaration per variable. For a better understanding please see [here](https://github.com/airbnb/javascript#variables--one-const)

Reply:
The const is for only one variable called avatars. Its content is an array of image files

Reply:
Seen. Thanks for clarity